Meniga/Meniga-ios-sdk/External/Sync/MNFSynchronization.h

```diff
@@ -62,7 +62,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 +(MNFJob *)synchronizeWithTimeout:(NSNumber *)timeout interval:(NSNumber *)interval completion:(MNFSynchronizationCompletionHandler)completion;
 
-+ (MNFJob*)synchronizeRealmUserWithId:(NSNumber *)realmUserId timeout:(NSNumber*)timeout interval:(NSNumber*)interval completin:(nullable MNFErrorOnlyCompletionHandler)completion;
++ (MNFJob*)synchronizeRealmUserWithId:(NSNumber *)realmUserId
+                         sessionToken:(nullable NSString *)sessionToken
+                              timeout:(NSNumber*)timeout
```

Comment:
Parameters are declared with different formatting (e.g. `NSNumber*` and `NSNumber *`). It would look clearer if it followed the same formatting.

Reply:
Formatted